ibis/backends/postgres/compiler.py

```diff
@@ -588,3 +588,26 @@ def visit_Cast(self, op, *, arg, to):
         return self.cast(arg, op.to)
 
     visit_TryCast = visit_Cast
+
+    def visit_Hash(self, op, *, arg):
+        arg_dtype = op.arg.dtype
+
+        if arg_dtype.is_int16():
+            return self.f.hashint2extended(arg, 0)
+        elif arg_dtype.is_int32():
+            return self.f.hashint4extended(arg, 0)
+        elif arg_dtype.is_int64():
+            return self.f.hashint8extended(arg, 0)
+        elif arg_dtype.is_float32():
+            return self.f.hashfloat4extended(arg, 0)
+        elif arg_dtype.is_float64():
+            return self.f.hashfloat8extended(arg, 0)
```

Comment:
Would all of these and also decimal types be covered by `hashnumericextended`?

Reply:
Oh, true...

```suggestion
        if arg_dtype.is_numeric():
            return self.f.hash_numeric_extended(arg, 0)
```